README.md

```diff
@@ -83,4 +83,20 @@ if __name__ == "__main__":
   opentracing.tracer.flush()
 ```
 
+### Thrift
+When using apache thrift rpc, make sure to both disable use_http by setting it to False as well
+as enabling use_thrift.
+
+```python
+return lightstep.Tracer(
+    component_name='WhatIDoWhatIDidnt',
+    access_token='587145376229b4e8cc4d99d948bb967b',
```

Comment:
Delete this access token and the collector host. This shouldn't be committed. In fact, I think to demonstrate this point you should do
```python
return lightstep.Tracer(
    ....,
    use_http=False,
    use_thrift=True,
    ....,
)
```

Reply:
Woops, sorry about that